packages/remirror__extension-entity-reference/src/entity-reference-extension.ts

```diff
@@ -103,6 +105,37 @@ export class EntityReferenceExtension extends MarkExtension<EntityReferenceOptio
     };
   }
 
+  /**
+   * Track click events passed through to the editor.
+   */
+  createEventHandlers(): CreateEventHandlers {
+    return {
+      /**
+       * listens to click events and call the "onClickMark" handler with any of:
+       * 1. no argument if the text clicked is not an highlight mark
+       * 2. the id of the clicked highlight mark
+       * 3. id of the shortest highlight mark in case of overlapping highlights
+       */
+      clickMark: (_event, clickState) => {
+        const { markRanges } = clickState;
```

Comment:
From what I know this returns all mark ranges, do we need to check if the mark type is actually "entity-reference" ?